libs/shared/ui/src/lib/components/modal/modal.tsx

```diff
@@ -94,6 +108,7 @@ export const Modal = (props: ModalProps) => {
           }}
           className="modal__overlay fixed left-0 top-0 flex h-screen w-full bg-neutral-700/20"
         />
+        {fakeModal && <div className="modal__overlay fixed left-0 top-0 flex h-screen w-full bg-neutral-700/20" />}
```

Comment:
You need to add an `onClick` to keep the outside click for modals

```suggestion
        {fakeModal && <div className="modal__overlay fixed left-0 top-0 flex h-screen w-full bg-neutral-700/20"
        onClick={(event) => {
              if (alertClickOutside) {
                event.preventDefault()
                setModalAlertOpen(true)
              } else {
                setExternalOpen ? setExternalOpen(false) : setOpen(false)
              }
            }} />}
```

It's the same `onClick` you have on the `<Dialog.Overlay>` maybe we need an function `onClickOverlay` something like this

Reply:
Fixed https://github.com/Qovery/console/pull/1386/commits/5be14c41a05acdf6d2bbd88a7fc4c152096826b2